jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -31,6 +31,14 @@
             description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
     scm:
         - git:
+            url: https://github.com/kubernetes/test-infra
+            basedir: go/src/k8s.io/test-infra
```

Comment:
That should be fine for PR Jenkins, but it's making me think that we should check both repos out to their own subdirectories.